### PR TITLE
Feat: Add Logo

### PR DESCRIPTION
**Pull request by Siphonophora**
Add logo.

### CODE REVIEW

**Comment by Siphonophora** [comment]
I checked the AppVeyor project and it has the icon file and the nuspec looks good too.

**Comment by Siphonophora** [comment]
@codito  Apparently I can't assign you as a reviewer on this project.

File: src/NUnit.Xml.TestLogger/NUnit.Xml.TestLogger.csproj [hunk 1]
```diff
@@ -17,4 +17,11 @@
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.5.0" />
     <PackageReference Include="System.ValueTuple" Version="4.3.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Stylecop.Analyzers" Version="1.1.118">
```
Comment (codito) [inline src/NUnit.Xml.TestLogger/NUnit.Xml.TestLogger.csproj]: Stylecop should be updated in scripts/dependencies.props

Comment (Siphonophora) [reply src/NUnit.Xml.TestLogger/NUnit.Xml.TestLogger.csproj]: Ah, got it. I was surprised to see stylecop was added to these csproj files but wasn't sure where else it should have happened.

**Review by codito — APPROVED** [review]

**Pull request merged**